doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -5,10 +5,10 @@
 X509_STORE_CTX_new_ex, X509_STORE_CTX_new, X509_STORE_CTX_cleanup,
 X509_STORE_CTX_free, X509_STORE_CTX_init, X509_STORE_CTX_set0_trusted_stack,
 X509_STORE_CTX_set_cert, X509_STORE_CTX_set0_crls,
-X509_STORE_CTX_get0_chain, X509_STORE_CTX_set0_verified_chain,
 X509_STORE_CTX_get0_param, X509_STORE_CTX_set0_param,
 X509_STORE_CTX_get0_untrusted, X509_STORE_CTX_set0_untrusted,
 X509_STORE_CTX_get_num_untrusted,
+X509_STORE_CTX_get0_chain, X509_STORE_CTX_set0_verified_chain,
```

Comment:
Why are these reordered?  IIUC the previous state was sorted alphabetically.

Reply:
Well, the functions were not really ordered alphabetically, and I believe a semantic ordering anyway makes more sense,
which had been the case more or less already.
So I've moved the description of those two functions just after `X509_STORE_CTX_get_num_untrusted` because they are strongly related.

Reply:
They should follow the order that is in the synopsis and description sections, right?

Reply:
Right.